doc/tools/meson.build

```diff
@@ -0,0 +1,93 @@
+sources = [
+    'cardos-tool.1.xml',
+    'dnie-tool.1.xml',
+    'dtrust-tool.1.xml',
+    'egk-tool.1.xml',
+    'eidenv.1.xml',
+    'goid-tool.1.xml',
+    'iasecc-tool.1.xml',
+    'openpgp-tool.1.xml',
+    'opensc-asn1.1.xml',
+    'opensc-explorer.1.xml',
+    'opensc-tool.1.xml',
+    'pkcs11-register.1.xml',
+    'pkcs11-tool.1.xml',
+    'pkcs15-crypt.1.xml',
+    'pkcs15-tool.1.xml'
+]
+
+if conf.get('ENABLE_OPENSSL')
+    sources += [
+        'cryptoflex-tool.1.xml',
+        'gids-tool.1.xml',
+        'netkey-tool.1.xml',
+        'piv-tool.1.xml',
+        'pkcs15-init.1.xml',
+        'sc-hsm-tool.1.xml',
+        'westcos-tool.1.xml'
+    ]
+endif
+
+if conf.get('ENABLE_OPENSSL') and conf.get('ENABLE_OPENPACE')
```

Comment:
Does meson support nested conditions?

I would find the nesting more readable than repeating the `conf.get('ENABLE_OPENSSL')` condition here.

Reply:
I prefer the opposite, but OK. I will change the pattern in src/tools/meson.build also.